### ISSUE TITLE
Generator uses a vulnerable module.

### ISSUE THREAD

**Issue by marshallswain** [donejs/generator-donejs#146]
Generated apps are showing up on the `Snyk` security vulnerability list because a deep dependency has a known seriousl threat.  https://snyk.io/test/npm/generator-donejs

We're not really affected by the threat, since we're only using it in the generator, but we probably want to update the module to get off the list.

**Comment by marshallswain** [comment]
We might need to update the plugin generator, too.

**Comment by matthewp** [comment]
This is gone from 1.0. 

<img width="423" alt="screen shot 2017-01-30 at 2 05 32 pm" src="https://cloud.githubusercontent.com/assets/361671/22437217/32aeacda-e6f5-11e6-8836-60e69be4c88a.png">

**Comment by phillipskevin** [comment]
I think Snyk is the website that reports vulnerabilities. minimatch 2.0 is the vulnerable module.

```
Vulnerable module: minimatch
Introduced through: yeoman-generator@0.20.3
Detailed paths

Introduced through: generator-donejs@1.0.0-alpha.8 › yeoman-generator@0.20.3 › findup-sync@0.2.1 › glob@4.3.5 › minimatch@2.0.10
```

**Comment by matthewp** [comment]
Ah, ok @phillipskevin if you are able to get the yeoman updated then hopefully that will resolve this.

**Comment by phillipskevin** [comment]
Yeah, should be fixed by https://github.com/donejs/generator-donejs/pull/181 once tests complete. `minimatch` is only used by devDependencies now:

```
$ npm ls minimatch
generator-donejs@1.0.0-alpha.8 /Users/kevin/dev/generator-donejs
├─┬ istanbul@0.4.5
│ └─┬ glob@5.0.15
│   └── minimatch@3.0.3
└─┬ mocha@2.5.3
  └─┬ glob@3.2.11
    └── minimatch@0.3.0
```

**Comment by phillipskevin** [comment]
I published this as 1.0.0-alpha.9, so I'll check this again once Snyk updates (not sure how long that takes).

**Comment by phillipskevin** [comment]
Looks good now: 
![image](https://cloud.githubusercontent.com/assets/5851984/22439871/dcc674e0-e6f6-11e6-853a-2a2f541cc0b9.png)
